js/client_exporter/xlsx/xlsx_cell_format_helper.js

```diff
@@ -1,6 +1,7 @@
 import typeUtils from "../../core/utils/type";
```

Comment:
You can import a single export from a module like below
`import { isDefined } from "../../core/utils/type";`

and replace `typeUtils.isDefined(fill)` with `isDefined(fill)`

Reply:
Sure. And, this change can make code a bit shorter in each file in the 'client_exporter\xlsx'.
Is it better to make this change in a separate PR?

Reply:
I think you can make it in this PR

Reply:
done